package.json

```diff
@@ -1,8 +1,8 @@
 {
   "name": "linter-pep8",
   "main": "./lib/main",
-  "version": "1.3.2",
-  "description": "Linter plugin for pep8",
+  "version": "1.4.0",
```

Comment:
The version number will get updated when it is released, so this needs to be taken out of the PR.